Source/WebCore/animation/TimelineRange.cpp

```diff
@@ -111,4 +225,14 @@ WTF::TextStream& operator<<(WTF::TextStream& ts, const SingleTimelineRange& rang
     return ts;
 }
 
+TimelineRange TimelineRange::defaultForScrollTimeline()
+{
+    return { { SingleTimelineRange::Name::Omitted, SingleTimelineRange::defaultValue(SingleTimelineRange::Type::Start) }, { SingleTimelineRange::Name::Omitted, SingleTimelineRange::defaultValue(SingleTimelineRange::Type::End) } };
```

Comment:
Would be easier to read if you added new lines and indented.
```suggestion
    return {
        { SingleTimelineRange::Name::Omitted, SingleTimelineRange::defaultValue(SingleTimelineRange::Type::Start) },
        { SingleTimelineRange::Name::Omitted, SingleTimelineRange::defaultValue(SingleTimelineRange::Type::End) }
    };
```

Reply:
Fixed.